psi4/src/psi4/libfock/points.cc

```diff
@@ -763,12 +767,12 @@ void BasisFunctions::compute_functions(std::shared_ptr<BlockOPoints> block) {
 
         // Copmute collocation
         if (deriv_ == 0) {
-            gg_collocation(L, npoints, x, y, z, nprim, norm, alpha, center.data(), (int)puream_, phi_start);
+            gg_collocation(L, npoints, xyz, 1, nprim, norm, alpha, center.data(), GG_SPHERICAL_GAUSSIAN, phi_start);
```

Comment:
This should be be `GG_SPHERICAL_GAUSSIAN` or `GG_CARTESIAN_ROW` I believe depending on `puream_`